### ISSUE TITLE
Apparent missing _Layout and associated files

### ISSUE THREAD

**Issue by Chas5555** [mguinness/IdentityManagerUI#9]
I think I have successfully integrated the IdentityManagerUI Razor class library into my project using your guidelines in the ReadMe.

However, when I  call /IdentityManager/home/users I get the following error: InvalidOperationException: The layout view '_Layout' could not be located.

I noticed that the re is no /Views/Shared/_Layout.cshtml file in the IdentityManagerUI code set.  When I install my own test /Views/Shared/_Layout.cshtml file the application works but without any styling on the pages.

Is there something missing in the download file: or have I missed something?

**Comment by mguinness** [comment]
It's been a little while since I last looked at this project, but the RCL doesn't include a _Layout.cshtml but instead uses it from the hosting web app under /Views/Shared.  Does you web app not already have a layout?

**Comment by Chas5555** [comment]
Thank you for the feedback.  I see what you mean.  It's because I am running Blazor Server as the main app so there isn't a  /Views/Shared folder.  I have researched this further and found a way of making it work with my Blazor app.  I used this article to make the changes and it works perfectly: [Including Static Resources In Razor Class Libraries In ASP.NET Core](https://www.mikesdotnetting.com/article/330/including-static-resources-in-razor-class-libraries-in-asp-net-core).  I also had to add a Nuget package to the RCL:  Microsoft.Extensions.FileProviders.Embedded.
